tasks/main.yml

```diff
@@ -1 +1,60 @@
 # role tasks
+
+- name: Check localectl status before
+  command: localectl status
+  register: localectl
+  changed_when: "localectl.rc != 0"
+
+- name: set the localectl before fact
+  set_fact:
+    localectl_before: "{{ localectl.stdout_lines | join ('\n') }}"
```

Comment:
`stdout_lines` is `stdout` split on newlines. Rather than joining `stdout_lines` with newlines, just use `stdout`.

Reply:
Also, this fact is unnecessary since the information being set as a fact is already contained in `localectl.stdout`.